### PR TITLE
overview page, dont display "https://"

### PR DESCRIPTION
**Pull request by larshp**
dont display "https://" or "http://" in overview page

delete unused method HAS_FAVORITES

part of #4528

### CODE REVIEW

**Review by mbtools — APPROVED** [review]
👍

File: src/ui/zcl_abapgit_gui_page_repo_over.clas.abap [hunk 1]
```diff
@@ -347,7 +337,10 @@ CLASS zcl_abapgit_gui_page_repo_over IMPLEMENTATION.
                                        iv_act = |{ c_action-select }?key={ <ls_overview>-key }| ) }</td>| ).
 
       IF <ls_overview>-type = abap_false.
-        ii_html->add( |<td>{ ii_html->a( iv_txt = <ls_overview>-url
+        lv_text = <ls_overview>-url.
+        REPLACE FIRST OCCURRENCE OF 'https://' IN lv_text WITH ''.
+        REPLACE FIRST OCCURRENCE OF 'http://' IN lv_text WITH ''.
+        ii_html->add( |<td>{ ii_html->a( iv_txt = lv_text
```
Comment (sbcgua) [inline src/ui/zcl_abapgit_gui_page_repo_over.clas.abap:343]: I'd probably add the full URL as a `iv_title`. For people to see if the connection is secured

Comment (larshp) [reply src/ui/zcl_abapgit_gui_page_repo_over.clas.abap:343]: true true, I'll add it and then merge to make room for https://github.com/abapGit/abapGit/pull/4541

**Comment by larshp** [comment]
title added in href

**Pull request merged**